### PR TITLE
Adding detection rule for esentutl utility

### PR DESCRIPTION
**Pull request by Sam0x90**
Used by Conti affiliates to target NTDS file and MSEdge info.

### CODE REVIEW

File: rules/windows/process_creation/Conti_esentutl.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+title: Detection of esentutl aka Extensible Storage Engine Utilities to gather credentials
```
Comment (frack113) [inline rules/windows/process_creation/Conti_esentutl.yaml:1]: `title: Gather Credentials with Esentutl Aka Extensible Storage Engine Utilities`
title don't start with detect and word are capitalization

File: rules/windows/process_creation/Conti_esentutl.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+title: Detection of esentutl aka Extensible Storage Engine Utilities to gather credentials
+id: 7df1713a-1a5b-4a4b-a071-dc83b144a101
+status: experimental
+author: sam0x90
+date: 2021/08/06
+description: Conti recommendation to its affiliates to use esentult to access NTDS dumped file. Trickbot also uses this utilities to get MSEdge info via its module "pwgrab"
+references:
+    - https://twitter.com/vxunderground/status/1423336151860002816
+	- https://attack.mitre.org/software/S0404/
```
Comment (frack113) [inline rules/windows/process_creation/Conti_esentutl.yaml:9]: Remove tab as it is a yaml file

File: rules/windows/process_creation/Conti_esentutl.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+tags:
+    - attack.credential_access
+    - attack.t1003
+    - attack.t1003.003
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        CommandLine|contains:
```
Comment (frack113) [inline rules/windows/process_creation/Conti_esentutl.yaml:20]: `CommandLine|contains: 'esentutl'`
Only 1 value no need a list

File: rules/windows/process_creation/Conti_esentutl.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+            - 'esentutl'
+    selection_password:
+        CommandLine|contains: ' /p'
+    condition: all of them
+falsepositives:
+    - To be determined
+level: medium
+fields:
+    - User
+	- CommandLine
```
Comment (frack113) [inline rules/windows/process_creation/Conti_esentutl.yaml:30]: Remove tab as it is a yaml file

File: rules/windows/process_creation/Conti_esentutl.yaml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        CommandLine|contains:
+            - 'esentutl'
+    selection_password:
+        CommandLine|contains: ' /p'
+    condition: all of them
```
Comment (frack113) [inline rules/windows/process_creation/Conti_esentutl.yaml:24]: Other way:
```yaml
   selection:
        CommandLine|contains|all:
            - 'esentutl'
            - ' /p'
    condition: selection
```
Fisrt way : `(event.code:"1" AND winlog.channel:"Microsoft\-Windows\-Sysmon\/Operational" AND process.command_line.keyword:*esentutl* AND process.command_line.keyword:*\ \/p*)`
Other way : alfer `(event.code:"1" AND winlog.channel:"Microsoft\-Windows\-Sysmon\/Operational" AND process.command_line.keyword:*esentutl* AND process.command_line.keyword:*\ \/p*)`
Same result

**Review by frack113 — CHANGES_REQUESTED** [review]

File: rules/windows/process_creation/process_susp_esentutl_params.yaml [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    - attack.t1003.003
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        CommandLine|contains|all:
+            - 'esentutl'
+            - ' /p'
+    condition: all of them
```
Comment (frack113) [inline rules/windows/process_creation/process_susp_esentutl_params.yaml:23]: `condition: selection`

**Review by frack113 — CHANGES_REQUESTED** [review]

**Comment by Sam0x90** [comment]
Thanks guys for the help!

**Review by frack113 — APPROVED** [review]

**Pull request merged**